src/cfe.h

```diff
@@ -5,7 +5,7 @@
  *  @author J. Ator
  *  @date 2017-11-16
  */
-#define MAX_MEANING_LEN 150
+#define MAX_MEANING_LEN 150 /**< ??? */
 
```

Comment:
Is the `<` character really needed here and in other similar definition comments?

In other words, does it really need to be `/**<` instead of just `/**` to start the comment?

Reply:
Ahh, OK, got it